cli/src/commands/unlock.ts

```diff
@@ -0,0 +1,35 @@
+import { CLIError } from '@oclif/errors';
+import Command from '../utils/command';
+import flags from '../utils/cli-flags';
+
+export default class Lock extends Command {
+  public static description = 'unlock an application';
+  public static examples = [
+    `$ ${Command.cliBinName} unlock`,
+    `$ ${Command.cliBinName} lock -s 'my-unlock-app' `,
```

Comment:
Did you mean unlock?
Please give a better app name here, something that look like a real deployed app name.